runtime/doc/options.txt

```diff
@@ -5603,6 +5609,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 			global
 	If in Insert, Replace or Visual mode put a message on the last line.
 	The |hl-ModeMsg| highlight group determines the highlighting.
+	The option is disabled if 'cmdheight' is 0.
```

Comment:
is this tested explicitly? something like:

```
set cmdheight=0
eq('showmode', 0)
set showmode  // error? or what?
```

Reply:
I have added the tests.